lib/install/management/store_files.go

```diff
@@ -276,7 +258,7 @@ func (d *Dispatcher) deleteVolumeStoreIfForced(conf *config.VirtualContainerHost
 		for label, url := range conf.VolumeLocations {
 			volumeStores.WriteString(fmt.Sprintf("\t%s: %s\n", label, url.Path))
 		}
-		log.Warnf("Since --force was not specified, the following volume stores will not be removed. Use the vSphere UI to delete content you do not wish to keep.\n%q", volumeStores.String())
+		log.Warnf("Since --force was not specified, the following volume stores will not be removed. Use the vSphere UI to delete content you do not wish to keep.\n%s", volumeStores.String())
```

Comment:
Hey @hickeng -- I remember in a PR seeing you tell Emma to use `%q` at the end of this log string, so I wanted to bring your attention to this and make sure it's OK that I replaced it with `%s`. When it was `%q` it was printing out like this:
`WARN[2016-08-10T12:08:50-05:00] Since --force was not specified, the following volume stores will not be removed. Use the vSphere UI to delete content you do not wish to keep.
"\tdefault: [datastore1] ians_volume_store_fo_sho\n" `

Now it looks like this:
`WARN[2016-08-10T12:29:07-05:00] Since --force was not specified, the following volume stores will not be removed. Use the vSphere UI to delete content you do not wish to keep.
	default: [datastore1] ians_volume_store_fo_sho
`